dojson/errors.py

```diff
@@ -26,6 +26,14 @@ class IgnoreKey(DoJSONException):
     """
 
 
+class IgnoreElem(DoJSONException):
```

Comment:
then we should call it `element` instead of `elem` however I still prefer `item`